paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
+  scientific computing with the earliest packages dating back into the 1970s.
+  Numerous open and proprietary software packages exist that aid users in
+  complex algebra, calculus, and other mathematical concepts that drive and
+  support both symbolic and numerical computing needs.
+  Here we present a symbolic CAS written in the Python programming language
+  that is developed by a large team of scientists, engineers, and software
+  developers in a bazaar open source development model.
```

Comment:
I would cite the original essay http://www.catb.org/~esr/writings/cathedral-bazaar/cathedral-bazaar/index.html

The book is a collection of essays.